cmd/kubeadm/app/cmd/config.go

```diff
@@ -123,7 +123,10 @@ func NewCmdConfigPrintDefault(out io.Writer) *cobra.Command {
 func getDefaultAPIObjectBytes(apiObject string) ([]byte, error) {
 	switch apiObject {
 	case constants.InitConfigurationKind, constants.MasterConfigurationKind:
-		return getDefaultInitConfigBytes()
+		return getDefaultInitConfigBytes(constants.InitConfigurationKind)
+
+	case constants.ClusterConfigurationKind:
+		return getDefaultInitConfigBytes(constants.ClusterConfigurationKind)
```

Comment:
It sounds a little bit strange to me calling `getDefaultInitConfigBytes` here.
What about implementing a separated `getDefaultClusterConfigBytes`?